src/app/styles/index.scss

```diff
@@ -88,3 +88,49 @@ figure {
     align-items: flex-start;
   }
 }
+
+.carousel {
```

Comment:
Why should we use it in globals styles instead component?

Reply:
Because this styles is written to override the library styles.

But inside a module file this styles would be hashed - `.carousel -> ._eq23_carousel`, the problem is that `._eq23_carousel` now is a different class hence not will override the libarry styles